src/sentry/notifications/notifications/base.py

```diff
@@ -44,6 +44,10 @@ def fine_tuning_key(self) -> str | None:
             return None
         return get_notification_setting_type_name(self.notification_setting_type)
 
+    @property
+    def from_email(self) -> str | None:
```

Comment:
Maybe I'm misunderstanding but isn't this the address we're emailing? So it's a `to_email`?

Reply:
@ceorourke No, it's actually the from email, see: https://github.com/getsentry/sentry/blob/master/src/sentry/utils/email/message_builder.py#L85

We can decide to send the email from a different address if we want, such as for billing emaills

Reply:
Oh ... interesting, I didn't know we did that. I thought this was the thing where you can specify a billing email address rather than needing to add them as a user.

Reply:
@ceorourke My bad if it wasn't clear, I'm talking about this email address: https://github.com/getsentry/getsentry/blob/master/getsentry/models/subscription.py#L679